spec/features/admin/non_admin_user_cannot_edit_user_as_admin_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature "Non admin user cannot edit user as admin" do
+  scenario "successfully" do
+    user = create(:user)
+    other_user = create(:user)
+    log_in_as user
+
+    visit admin_users_path(other_user)
+
+    expect(page).to have_text(I18n.t("admin.defaults.require_admin"))
```

Comment:
Once again, not needed. The more feature specs we add, the slower the test suite will take to run, so we try and have fewer feature specs (which is why we test for that functionality elsewhere)

Reply:
Done in sha: 33f2dfc